tests/alt-registry.rs

```diff
@@ -65,11 +65,13 @@ fn depend_on_alt_registry() {
     // Don't download a second time
     assert_that(p.cargo("build").masquerade_as_nightly_cargo(),
                 execs().with_status(0).with_stderr(&format!("\
+[UPDATING] registry `{reg}`
```

Comment:
Hm this looks like a bug? I don't think that an update should happen here for a clean build?